### DISCUSSION TITLE
why must enter the password of the remote (Windows) server

### DISCUSSION THREAD

**Discussion by ztyandjw** [rustdesk/rustdesk#7357] [General]
why should  enter the password of the remote (Windows) server for the first time. How can I avoid this operation?
Using other remote desktop solutions, there is no requirement to input the operating system password

![111](https://github.com/rustdesk/rustdesk/assets/14125652/d95ade09-88a1-4630-a6cc-0aa0d2ba2b64)

**Comment by xlionjuan** [comment]
Uhhh, it is normal, most remote desktop software will capture the current screen
